src/main.rs

```diff
@@ -212,6 +215,29 @@ fn main() -> Result<(), Error> {
 	if config.get_option::<String>("sensors/#0/loc").is_some() {
 		let mut environment = Environment::new(&mut config);
 
+		let mut ir_temp = ModIrTemps::new_default();
```

Comment:
Good placement but the constructor's name is different. Please also be consistent in singular name of modules. I think ModIR is enough. Pass config to constructor as for other modules.

```suggestion
		let mut ir_temp = ModIR::new(&mut config);
```